Firestore/core/test/firebase/firestore/util/string_util_test.cc

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google
+ * Copyright 2018 Google
```

Comment:
The copyrights on these moved files can remain 2017.

Reply:
done